tslint.json

```diff
@@ -1,4 +1,8 @@
 {
+  "tslint.options": {
```

Comment:
@domoritz  Can you explain why we need this part? (It seems irrelevant to this PR?)

Reply:
This just enables better linting is vscode because it knows the arguments we pass to tslint.